gazebo/transport/TaskGroup.hh

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2021 Alex Dewar
```

Comment:
is it possible to assign copyright to Open Source Robotics Foundation but add your name to the `AUTHORS` file and in a `Changelog` entry?

Reply:
Sure!

Reply:
thank you ❤️